py/ai/h2o/sparkling/ml/__init__.py

```diff
@@ -0,0 +1,16 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Just asking... is this `__init__` file sufficient? We don't need init files in parent folders?

Reply:
__init__ file needs at every level, I checked the current state of this PR and seems it is available at all levels